src/Mod/Cloud/Gui/Workbench.h

```diff
@@ -0,0 +1,47 @@
+/***************************************************************************
+ *   Copyright (c) 2019         jmverdun3@gmail.com                        *

```

Comment:
please add you name before the email and fix whitespace after it

Reply:
shall be fixed